### PR TITLE
Clarify Problem Solving process description in README

### PR DESCRIPTION
**Pull request by codesections**
This is an attempted solution to #272 as described in that issue.  In particular, it is intended to clarify the existing process without adding any substantive changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,104 +4,153 @@
 that require discussion and/or consensus. This document describes the
 process in more detail.
 
+## The Problem-Solving Process
+### Step 1: Reporting a problem
 
-## Reporting a problem
-
-Anyone is welcome to create a new issue. The ticket should *only*
+Anyone is welcome to create a new issue. The issue should *only*
```
Comment (JJ) [inline README.md:10]: Maybe clarify that "reporting a problem" implies "creating a new issue". Something like "anyone is welcome to report a problem by creating a new issue"

File: README.md [hunk 2]
```diff
@@ -4,104 +4,153 @@
...
+need to be worked out).  As a more concrete example, bugs in the 
+Rakudo compiler should be reported in
 [rakudo repository](https://github.com/rakudo/rakudo/issues).
 
-## Initial proposal
+If someone opens an issue that doesn't belong in Problem Solving,
+then anyone with sufficient GitHub permissions to do so should close
+the issue and direct the user to the correct place to address that 
+issue (e.g., the Rakudo repo).
+
```
Comment (JJ) [inline README.md:29]: Or move it themselves, if they have the privs (most of us do)

**Review by JJ — APPROVED** [review]
My comments are mostly nits. It's, at large, a good proposal. Thanks a lot.

File: README.md [hunk 3]
```diff
@@ -4,104 +4,153 @@
...
-heroless ideas.
+By proposing a solution, you are _typically_ volunteering to implement 
+that solution.  If you know that you won't be able to work on a 
+full solution, please **say so when proposing that solution**. Having a 
+hero who will carry a solution though to implementation is often 
+essential, but it is also  important to hear out good heroless ideas.
+
+### Step 3: Discussion of the proposal
 
+After the initial proposal, everyone should discuss the idea and point
```
Comment (vrurg) [inline README.md:51]: I'd say, "should" is a bit too much. "may", perhaps?

File: README.md [hunk 4]
```diff
@@ -4,104 +4,153 @@
...
+out any flaws, implementation details, suggested changes, etc. that 
+they might see.  (Of course, both before and during step 3, people 
+can discuss the problem generally rather than a particular solution.)
 
-### Assignees and their role
+In step 3, anyone can comment, but two people have as especially
```
Comment (japhb) [inline README.md:56]: two people have *an* especially

**Review by JJ — APPROVED** [review]
Thanks!

**Review by lizmat — APPROVED** [review]

**Review by vrurg — APPROVED** [review]

**Review by rba — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -48,7 +48,7 @@ essential, but it is also  important to hear out good heroless ideas.
 
 ### Step 3: Discussion of the proposal
 
-After the initial proposal, everyone may discuss the idea and point
+After the initial proposal, everyone is encoraged to discuss the idea and point
```
Comment (japhb) [inline README.md]: "encouraged"

Comment (JJ) [reply README.md]: That was fixed a bit later :+1:

**Review by FCO — APPROVED** [review]

**Pull request merged**